internal/grader/test.go

```diff
@@ -41,6 +42,10 @@ func GetTestSubmissions(baseDir string) ([]*TestSubmissionInfo, error) {
 			return nil, fmt.Errorf("Could not find assignment for test submission '%s': '%w'.", testSubmissionPath, err)
 		}
 
+		if assignment.ID != "bash" && config.DOCKER_DISABLE.Get() {
```

Comment:
I don't think `config.DOCKER_DISABLE` would be the thing to check here.
What about a user that is running the tests on a machine where docker is runner?
We want them the have the same experience as someone who does not have docker running (i.e., the same set of tests should run for TestNoDockerSubmissions in both cases).

Instead, this function should probably take a boolean (`useDocker`?) that tells this function if we want those other test cases.